src/nupic/os/Timer.cpp

```diff
@@ -66,38 +66,6 @@ static UInt64 getCurrentTime()
   return (UInt64)(v.QuadPart) - initialTicks_;
 }
 
-#elif defined(NTA_PLATFORM_darwin)
```

Comment:
@scottpurdy There's not bug longer. The bug arose because I revived code that shoudn't be revived. Darwin always used the same code than Linux which is under `#else` block (https://github.com/numenta/nupic.core/blame/master/src/nupic/os/Timer.cpp#L101). It seems that the code under `NTA_PLATFORM_darwin` never was used and this block exists since April/2013:

https://github.com/numenta/nupic.core/blame/master/src/nupic/os/Timer.cpp#L69

It's simply a dead code that I shound't have revived.